Added a link to the GitHub repository.

phistuck opened the pull request:
Decreased the footer anchor margin and added a link to the GitHub repository.

phistuck committed:
Added a link to the GitHub repository.

There is no GitHub link in the page, so a link was added for better discovery.

phistuck committed:
Made some room for another footer link.

Reduced the padding of the footer links, in preparation for the new GitHub link.

jacobrossi commented:
Thanks for the pull request! Good addition to make. Can you please revise this to target the development branch?  That's where we stage our changes before deployment (I'll get that added to the Readme so others know).

phistuck committed:
Updated the GitHub link.

The GitHub link referred to the default branch, so it was changed to the development branch, by request of Jacob Rossi.

phistuck commented:
Why not use the default branch?
Anyway, done, I guess?

jacobrossi commented:
Thanks again for this. Looks like I was confusing with my comment. I meant just request the PR be merged into dev instead of prod (we have a testing server that auto deploys from the development branch for final testing before going live). You're correct that the link should go to the default branch. So I reverted the href change and just merged this into dev on my machine.
